docs/source/cmdline/delete.rst

```diff
@@ -19,7 +19,7 @@ Options
 
     Force deletion without requesting interactive confirmation
 
-.. option:: -i, --instance <instance-name>
+.. option:: -i, --dandi-instance <instance-name>
```

Comment:
just to make sure -- you are not changing API here, and rather just fixing up documentation right? (i.e. that it was `dandi_instance` already)

Reply:
Correct.